hrjobcontract/CRM/Hrjobcontract/BAO/HRJobHealth.php

```diff
@@ -56,6 +56,35 @@ public static function create($params) {
 
 
   /**
+   * Check insurance provider if exist   .
+   *
+   * @param String $searchValue
+   * @param String $providerType
+   * @return Integer ( Provider ID or 0 if not exist)
+   */
+  public static function checkProvider($searchValue, $providerType) {
+    if (is_numeric ($searchValue))  {
+      $searchField = 'id';
+    }
+    else {
+      $searchField = 'display_name';
+    }
+    $queryParam = array(1 => array($searchValue, 'String'));
+    $query = "SELECT id,contact_sub_type FROM civicrm_contact WHERE ".$searchField." = %1 " .
```

Comment:
I think we should use `"SELECT id,contact_sub_type FROM civicrm_contact WHERE {$searchField}"` instead of `"SELECT id,contact_sub_type FROM civicrm_contact WHERE ".$searchField."`. Nothing programatically wrong with this but it just decreased the number of double quotes :)